src/main/java/com/cvsgo/service/UserService.java

```diff
@@ -85,4 +95,52 @@ public boolean isDuplicatedEmail(String email) {
     public boolean isDuplicatedNickname(String nickname) {
         return userRepository.findByNickname(nickname).isPresent();
     }
+
+    /**
+     * 회원 팔로우를 생성한다.
+     *
+     * @param user   로그인한 사용자
+     * @param userId 팔로잉할 사용자 ID
+     * @throws NotFoundUserException         해당하는 아이디를 가진 사용자가 없는 경우
+     * @throws BadRequestUserFollowException 본인을 팔로우하는 경우
+     * @throws DuplicateUserFollowException  이미 해당하는 회원 팔로우가 존재하는 경우
+     */
+    @Transactional(rollbackFor = Exception.class)
+    public void createUserFollow(User user, Long userId) {
+        User followingUser = userRepository.findById(userId).orElseThrow(() -> NOT_FOUND_USER);
+        if (user.equals(followingUser)) {
+            throw BAD_REQUEST_USER_FOLLOW;
+        }
+
+        UserFollow userFollow = UserFollow.create(followingUser, user);
+        try {
+            userFollowRepository.save(userFollow);
+        } catch (DataIntegrityViolationException e) {
+            entityManager.clear();
+            if (Boolean.TRUE.equals(
+                userFollowRepository.existsByUserAndFollower(followingUser, user))) {
+                log.info("중복된 회원 팔로우: {} '{}'", user, followingUser);
```

Comment:
User 클래스에 toString() 메소드가 오버라이딩 되어있지 않아서
`중복된 회원 팔로우: com.cvsgo.entity.User@3f70a3a9 'com.cvsgo.entity.User@402d87d8` 이렇게 보일 것 같아요.
이렇게 닉네임으로 로그를 찍으면 어떨까요?
```suggestion
                log.info("중복된 회원 팔로우: {} '{}'", user.getNickname(), followingUser.getNickname());
```

Reply:
아하 그러네요 그렇다면.. id 로그를 찍는 것으로 바꾸겠습니다!!